### DISCUSSION TITLE
Problem when importing hosts from checkmk

### DISCUSSION THREAD

**Discussion by MajaSmereczynska** [kuhn-ruess/cmdbsyncer#27] [Q&A] [answered]
Checkmk Raw Edition 2.2.0p7
CMDB Syncer 3.2.3
Netbox v3.5.1

I am using the syncer to import hosts from checkmk and export them to netbox. Since the upgrade to version 3.2.3 when importing the hosts they show up in the syncer correctly but the Available, Last Last Import Seen and Last Import Sync fields are empty. I am not sure if that is the problem but when I try to export the hosts to netbox it doesn't work. I don't get an error or anything, it's as if the syncer didn't see those hosts at all.

**Comment by Bastian-Kuhn** [comment]
Hi, 

1) Are they whitelisted?
2) What is shown with ./cmdbsyncer netbox debug_host HOSTNAME
3) Have you hit "Commit Changes" in Gui? 

Last Import Seen I need to fix, will push that asap

**Reply by MajaSmereczynska** [reply]
1. What do you mean by whitelisted? I only found the option to whitelist an attribute in Rules -> Checkmk -> Filter
2. 
![Screenshot 2023-07-27 153547](https://github.com/kuhn-ruess/cmdbsyncer/assets/83644808/80cacd2e-ce88-45bb-8759-d631277cdbcb)
3. I have yes and it didn't help

**Reply by Bastian-Kuhn** [reply, marked as answer]
I just started up a new Netbox Instance, then tested the workflow from importing from Checkmk and then exporting to net box.

Besides the initial installation, like having a Fallback Site, or have the cmdbsyncer_id Field which needs to be created, I can't see any problem?

I improved the errors a bit. Can you pull the latest git version, enable the Debug log like in this: https://cmdbsyncer.readthedocs.io/en/latest/basics/lcl_config/ example, and show me what is returned?

**Reply by MajaSmereczynska** [reply]
I pullled the latest version and now it is working again. Thank you.

**Reply by Bastian-Kuhn** [reply]
That's strange, I changed nothing but log outputs and handling.
